services/category.js

```diff
@@ -15,6 +15,12 @@ const categoryService = {
     return await Category.findById(id).lean().exec()
   },
 
+  addCategory: async (data) => {
+    const { name, categoryIcon } = data
+
```

Comment:
add capitalizing for the first letter if it's needed (look into subject service)

Reply:
fixed